open_academy/controllers/__init__.py

```diff
@@ -0,0 +1,3 @@
+# -*- coding: utf-8 -*-
+
+from . import controllers
```

Comment:
```suggestion
from . import controllers
```
Add newline at the end